netbox_devicetype_importer/models.py

```diff
@@ -1,26 +1,28 @@
 from django.db import models
 
 from utilities.querysets import RestrictedQuerySet
+from .choices import TypeChoices
 
 
 class MetaDeviceType(models.Model):
-    name = models.CharField(max_length=100)
-    vendor = models.CharField(max_length=50)
-    sha = models.CharField(max_length=40)
-    download_url = models.URLField(null=True, blank=True)
-    is_new = models.BooleanField(default=True)
-    imported_dt = models.IntegerField(null=True, blank=True)
-    is_imported = models.BooleanField(default=False)
+	name = models.CharField(max_length=100)
+	vendor = models.CharField(max_length=50)
+	type = models.CharField(max_length=20, choices=TypeChoices, default=TypeChoices.TYPE_DEVICE)
+	sha = models.CharField(max_length=40)
+	download_url = models.URLField(null=True, blank=True)
+	is_new = models.BooleanField(default=True)
+	imported_dt = models.IntegerField(null=True, blank=True)
+	is_imported = models.BooleanField(default=False)
 
-    objects = RestrictedQuerySet.as_manager()
```

Comment:
I am little confused about indents. What do you use for code formatting?

Reply:
It's the default of `Pycharm Professional` since there was no `.editorconfig` was present in the repository

Reply:
you can add this file if formatting depends on it, but don't forget to add this filename to .gitignore

Reply:
anyway you have to fix intends

Reply:
I'm not sure what you mean by this. I would suggest you provide me with the formatting which you would like me to use. I would prefer to use your repo style instead of mine.

References: 

- https://editorconfig.org/
- https://github.com/django/django/blob/main/.editorconfig

Reply:
just use 4 space as indent

Reply:
I use [black](https://github.com/psf/black) formatter

Reply:
All fixed as per your request